docs/examples.rst

```diff
@@ -45,4 +45,4 @@ More advanced examples doing multiple things at the same time.
 
    examples/2-advanced/cosmic_shear.ipynb
    examples/2-advanced/stage_4_galaxies.ipynb
-   examples/2-advanced/legacy-mode.ipynb
+   examples/2-advanced/legacy_mode.ipynb
```

Comment:
I like the dash more, no need to rename it

Reply:
and there are already any number of links to https://glass.readthedocs.io/stable/examples/2-advanced/legacy-mode.html floating around various talks and slides

Reply:
Okay, reverted